homeassistant/components/image_processing/__init__.py

```diff
@@ -182,14 +182,14 @@ def state_attributes(self):
         """Return device specific state attributes."""
         return {ATTR_FACES: self.faces, ATTR_TOTAL_FACES: self.total_faces}
 
-    def process_faces(self, faces, total):
+    def process_faces(self, faces: list[dict[str, Any]], total: int) -> None:
         """Send event with detected faces and store data."""
         run_callback_threadsafe(
             self.hass.loop, self.async_process_faces, faces, total
         ).result()
 
     @callback
-    def async_process_faces(self, faces, total):
+    def async_process_faces(self, faces: list[dict[str, Any]], total: int) -> None:
```

Comment:
This could be a Type Alias with a non-total TypedDict. The keys seem to be well defined.

Reply:
done